src/PowerShellEditorServices/Services/TextDocument/Handlers/TextDocumentHandler.cs

```diff
@@ -91,11 +89,9 @@ public Task<Unit> Handle(DidOpenTextDocumentParams notification, CancellationTok
                     notification.TextDocument.Uri,
                     notification.TextDocument.Text);
 
-#pragma warning disable CS4014
             // Kick off script diagnostics without blocking the response
             // TODO: Get all recently edited files in the workspace
-            _analysisService.RunScriptDiagnosticsAsync(new ScriptFile[] { openedFile });
-#pragma warning restore CS4014
+            _analysisService.RunScriptDiagnostics(new ScriptFile[] { openedFile }, token);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the array type; it is redundant.](https://app.codacy.com/manual/TylerLeonhardt/PowerShellEditorServices/pullRequest?prid=4961943)